applications/RomApplication/tests/fluid_dynamics_test_files/LSPGROM/ProjectParametersLSPGROM.json

```
@@ -0,0 +1,98 @@
+{
+    "analysis_stage": "KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis",
+    "problem_data"     : {
+        "problem_name"  : "couette_flow_test",
+        "parallel_type" : "OpenMP",
+        "echo_level"    : 0,
+        "start_time"    : 0.0,
+        "end_time"      : 100.0
+    },
+    "output_processes" : {
+        "gid_output" : [],
+        "vtk_output" : []
+    },
+    "solver_settings"  : {
+        "model_part_name"             : "FluidModelPart",
+        "domain_size"                 : 2,
+        "solver_type"                 : "Monolithic",
+        "model_import_settings"       : {
+            "input_type"     : "mdpa",
+            "input_filename" : "../../../../FluidDynamicsApplication/tests/CouetteFlowTest/couette_flow_test"
+        },
+        "material_import_settings"    : {
+            "materials_filename" : "../../../../FluidDynamicsApplication/tests/CouetteFlowTest/FluidMaterials.json"
+        },
+        "relative_velocity_tolerance" : 1e-6,
+        "absolute_velocity_tolerance" : 1e-8,
+        "relative_pressure_tolerance" : 1e-6,
+        "absolute_pressure_tolerance" : 1e-8,
+        "volume_model_part_name"      : "FluidParts_Fluid",
+        "skin_parts"                  : ["AutomaticInlet2D_Inlet","Outlet2D_Outlet","NoSlip2D_Bottom","VelocityConstraints2D_Top"],
+        "time_scheme"                 : "bossak",
+        "time_stepping"               : {
+            "automatic_time_step" : false,
+            "time_step"           : 10.0
+        },
+        "formulation"                 : {
+            "element_type"             : "vms"
```

```suggestion
            "element_type"             : "qsvms"
```
The `vms` element is the legacy one. I'd use the `qsvms` which implements the very same formulation according to current standards.